### PR TITLE
NAS-113406 / 22.02 / NAS-113406: Change FreeBSD wording

### PR DESCRIPTION
**Pull request by undsoft**

### CODE REVIEW

File: src/app/helptext/network/interfaces/interfaces-form.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default {
   ],
 
   int_interface_placeholder: T('NIC'),
-  int_interface_tooltip: T('Enter the FreeBSD device name of the\
+  int_interface_tooltip: T('Enter the device name of the\
  interface. This cannot be changed after the interface is created.'),
   int_interface_validation: [Validators.required],
```
Comment (RehanY147) [inline src/app/helptext/network/interfaces/interfaces-form.ts:29]: What about place where links from www.freebsd.org are used? Are those alright?

**Review by RehanY147 — APPROVED** [review]
Looks good to me otherwise.

File: src/app/helptext/network/interfaces/interfaces-form.ts (hunk 1 above)
Comment (undsoft) [reply src/app/helptext/network/interfaces/interfaces-form.ts:29]: Addressed in another PR.

File: src/assets/i18n/af.json [hunk 2]
```diff
@@ -506,7 +506,7 @@
   "Browse to the keytab file to upload.": "",
   "Browse to the locally saved configuration file.": "",
   "Browse to the operating system installer image file.": "",
-  "Browse to the path to be copied. FreeBSD  file path limits apply. Other operating systems can have different  limits which might affect how they can be used as sources or  destinations.": "",
+  "Browse to the path to be copied. Linux  file path limits apply. Other operating systems can have different  limits which might affect how they can be used as sources or  destinations.": "",
```
Comment (Mrt134) [inline src/assets/i18n/af.json:509]: Found these redundant empty spaces. I see the same in the other i18n files - is this an error from translation file generation?
```suggestion
  "Browse to the path to be copied. Linux file path limits apply. Other operating systems can have different limits which might affect how they can be used as sources or destinations.": "",
```

**Review by Mrt134 — COMMENTED** [review]
Found one small thing.

File: src/assets/i18n/af.json (hunk 2 above)
Comment (undsoft) [reply src/assets/i18n/af.json]: It's an artifact of how the code was written.

**Pull request merged**